app/lib/pathfinder_provisioner.rb

```diff
@@ -10,10 +10,9 @@ def initialize(pathfinder_host,
     @pathfinder_host = pathfinder_host
     @pathfinder_token = pathfinder_token
     @pathfinder_cluster = pathfinder_cluster
-    @image = opts[:image] || '18.04'
   end
 
-  def provision!(hostname)
+  def provision!(hostname, image = "18.04")
```

Comment:
we should put this default value (`18.04`) somewhere else, either database or environment variable

Reply:
Done https://github.com/BaritoLog/BaritoMarket/commit/950012cff68de4c44e26a136a4445ed63c34dfdf

since all infra component default value come from component template, I added image default on component template.